src/index.test.js

```diff
@@ -53,44 +56,44 @@ test.beforeEach(t => {
 
 test('renders default state', t => {
   const { tree } = t.context
-  const wrapper = mount(<DropdownTreeSelect data={tree} />)
+  const wrapper = mount(<DropdownTreeSelect id={dropdownId} data={tree} />)
```

Comment:
I don't think we need to update all the existing tests with id. To ensure that everything is backward compatible, we need to make sure that not passing the new prop is not breaking anything. It should autogenerate one (id), if not passed, correct? Correct me if I'm wrong here, but I'd imagine that the existing tests will have autogenerated id in their snapshot, and an additional test that passes `id` to ensure correctness of the new implementation.

Reply:
I've updated all snapshots with id to not rely on order they're executed. There's a separate test to ensure unique ids.